test/functionalTest/cases/3693_covered_notifications_in_subscritpions/covered_notifications_in_subscritpions.test

```diff
@@ -0,0 +1,691 @@
+            "attrs": [
+                "pressure"
+            ]
+        },
+        "entities": [
+            {
+                "id": "Room2",
+                "type": "Room"
+            }
+        ]
+    }
+}
+
+
+09. PATCH /v2/entities/Room2/attrs, to update the value of pressure attribute.
+==============================================================================
+HTTP/1.1 204 No Content
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+10. Dump the accumulator, see the missing attribute in the notification added with value null and type NONE.
+============================================================================================================
+POST http://localhost:9997/notify
+Fiware-Servicepath: /
+Content-Length: 249
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}); cbnotif=1
+
+{
+    "data": [
+        {
+            "brightness": {
+                "metadata": {},
+                "type": "None",
+                "value": null
+            },
+            "id": "Room2",
+            "pressure": {
+                "metadata": {},
+                "type": "Float",
+                "value": 740
+            },
+            "temperature": {
+                "metadata": {},
+                "type": "Float",
+                "value": 24
+            },
+            "type": "Room"
+        }
+    ],
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+11. POST /v2/entities, to create Room3 with temperature and pressure.
+=====================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/Room3?type=Room
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+12. POST /v2/subscriptions, to create subscription with covered=false and some attributes in attrs which is not present in entities.
+====================================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+13. GET /v2/subscriptions/subscription_id, to check the covered=false.
+======================================================================
+HTTP/1.1 200 OK
+Content-Length: 358
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "A subscription to get info about Room3",
+    "id": "REGEX([0-9a-f]{24})",
+    "notification": {
+        "attrs": [
+            "temperature",
+            "humudity",
+            "brightness"
+        ],
+        "attrsFormat": "normalized",
+        "http": {
+            "url": "http://localhost:9997/notify"
+        },
+        "onlyChangedAttrs": false
```

Comment:
covered is pretty similar to onlyChangedAttrs and should be treated the same way. I mean, if it is not included in the subscription creation request, it should be added as `"covered": false`.

This would involve modifying some existing .test (as we did for onlyChangedAttrs in PR https://github.com/telefonicaid/fiware-orion/pull/3490), but I think it is affordaable.

Reply:
Fixed in f4a5cac69e31bf5392c5c327f90502e9ec59aaf8